src/validate/js/propValidators/computed.ts

```diff
@@ -22,6 +25,23 @@ export default function computed(validator: Validator, prop: Node) {
 	checkForComputedKeys(validator, prop.value.properties);
 
 	prop.value.properties.forEach((computation: Node) => {
+		const name = getName(computation.key);
+
+		if (!isValidIdentifier(name)) {
+			const suggestion = name.replace(/[^_$a-z0-9]/ig, '_').replace(/^\d/, '_$&');
```

Comment:
should this be moved to a helper?

Reply:
I tend to favour leaving things like this inline until you need them in more than one place, otherwise I find you can end up with a lot of premature abstractions